META.coq

```diff
@@ -20,7 +20,7 @@ package "clib" (
   version     = "8.8"
 
   directory   = "clib"
-  requires    = "str, unix, threads"
+  requires    = "num, str, unix, threads"
```

Comment:
How's this change related to this PR?

Reply:
Likely was needed for proper testing; let's split to its corresponding OR.